pylint/pyreverse/inspector.py

```diff
@@ -194,24 +171,7 @@ def visit_classdef(self, node: nodes.ClassDef) -> None:
                 if not isinstance(assignattr, nodes.Unknown):
                     self.associations_handler.handle(assignattr, node)
                     self.handle_assignattr_type(assignattr, node)
-        # resolve implemented interface
-        try:
-            ifaces = interfaces(node)
-            if ifaces is not None:
-                node.implements = list(ifaces)
-                if node.implements:
-                    # TODO: 3.0: Remove support for __implements__
-                    warnings.warn(
-                        "pyreverse will drop support for resolving and displaying "
-                        "implemented interfaces in pylint 3.0. The implementation "
-                        "relies on the '__implements__'  attribute proposed in PEP 245"
-                        ", which was rejected in 2006.",
-                        DeprecationWarning,
-                    )
-            else:
-                node.implements = []
-        except astroid.InferenceError:
-            node.implements = []
+        node.implements = []
```

Comment:
Can't we just remove this attribute as well?

Reply:
I was wondering also if you were planning on removing all ``IChecker`` classes ?

Reply:
No, they just don't need to be `Ichecker` anymore

Reply:
Oh wait, they do need to be. So no, they should remain as is.

Reply:
Wait.... hahah Which `IChecker` classes are you referring to?